source/solvers/navier_stokes_base.cc

```diff
@@ -1266,6 +1247,37 @@ NavierStokesBase<dim, VectorType, DofsType>::refine_mesh_uniform()
     this->update_multiphysics_time_average_solution();
 }
 
+template <int dim, typename VectorType, typename DofsType>
+void
+NavierStokesBase<dim, VectorType, DofsType>::transfer_without_refinement()
```

Comment:
Why do we need this? If we rotate the mesh, the vectors do not changes. So there is no need for transfer.

Reply:
There's no need to transfer that's true. Only the boundary conditions have to be updated.